### ISSUE TITLE
delete-old-deployments tries to delete current deployment twice

### ISSUE THREAD

**Issue by juhmelo** [zalando-incubator/zalando-deploy-cli#9]
```
zdeploy delete-old-deployments nginx-test asdfgh 28 --execute
Deleting deployment nginx-test-asdfgh-28..
Deleting deployment nginx-test-asdfgh-28..
```

**Comment by hjacobs** [comment]
I could now reproduce the problem :disappointed:

**Comment by hjacobs** [comment]
Should be fixed with #10
